<table>
  <thead>
    <tr>
      <th>Tool Name</th>
      <th>Description</th>
      <th>Link</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>EMBL-SMART</td>
      <td>Simple Module Architecture Tool </td>
      <td><a class="btnFormating" href="http://smart.embl-heidelberg.de/">Link</a></td>
    </tr>
    <tr>
      <td>SWISS MODEL</td>
      <td>This server's goal is to make protein modelling available to all life scientists worldwide</td>
      <td><a class="btnFormating" href="https://swissmodel.expasy.org/interactive">Link</a></td>
    </tr>
    <tr>
      <td>1000 Genome Browser</td>
      <td>Users can explore variant calls, genotype calls, and supporting data generated by the 1000 Genomes Project using this interactive graphical viewer</td>
      <td><a class="btnFormating" href="http://asia.ensembl.org/Homo_sapiens/Info/Index">Link</a></td>
    </tr>
    <tr>
      <td>UCSC-Genome Browser</td>
      <td>It enables scientists to observe all 23 human chromosomes at any scale, from an entire chromosome down to a single nucleotide</td>
      <td><a class="btnFormating" href="https://genome.ucsc.edu/">Link</a></td>
    </tr>
    <tr>
      <td>LVariant Annotation Integrator</td>
      <td>It is a research tool for connecting your submitted set of variant calls with annotations from the UCSC database. To forecast the functional effects of variations on transcripts, it uses gene annotations</td>
      <td><a class="btnFormating" href="https://genome.ucsc.edu/cgi-bin/hgVai">Link</a></td>
    </tr>
    <tr>
      <td>Genome Browser</td>
      <td>Users can view many kinds of sequence-associated data in a genomic context</td>
      <td><a class="btnFormating" href="https://genome.ucsc.edu/cgi-bin/hgGateway">Link</a></td>
    </tr>
    <tr>
      <td>Data Integrator</td>
      <td>LorBy determining which elements in several tracks overlap one another, the Data Integrator can output all fields from all chosen tracks. You can query up to 5 distinct tracks at once</td>
      <td><a class="btnFormating" href="https://genome.ucsc.edu/cgi-bin/hgIntegrator">Link</a></td>
    </tr>
    <tr>
      <td>BLAST</td>
      <td>It looks for areas where biological sequences resemble one another. The application computes the statistical significance by comparing nucleotide or protein sequences to sequence databases</td>
      <td><a class="btnFormating" href="https://www.uniprot.org/blast">Link</a></td>
    </tr>
    <tr>
      <td>Variant Effect Predictor</td>
      <td>Variant Effect Predictor (VEP) tool, analyse variants and forecast the functional effects of known and hypothetical variants</td>
      <td><a class="btnFormating" href="http://asia.ensembl.org/info/docs/tools/index.html">Link</a></td>
    </tr>
    <tr>
      <td>Variant Recoder</td>
      <td>Transform a variant identifier into all conceivable variant IDs in HGVS, VCF, or genomic SPDI notation.</td>
      <td><a class="btnFormating" href="http://asia.ensembl.org/Homo_sapiens/Tools/VR">Link</a></td>
    </tr>
    <tr>
      <td>BLAST/BLAT</td>
      <td>Find DNA or protein sequence in genomes</td>
      <td><a class="btnFormating" href="http://asia.ensembl.org/Homo_sapiens/Tools/Blast">Link</a></td>
    </tr>
    <tr>
      <td>File Chameleon</td>
      <td>IEnsembl files should be converted for usage with other analysis tools.</td>
      <td><a class="btnFormating" href="http://asia.ensembl.org/Homo_sapiens/Tools/FileChameleon">Link</a></td>
    </tr>
    <tr>
      <td>Assembly Converter</td>
      <td>IEnsData's coordinates will be mapped (liftover) to the present assembly</td>
      <td><a class="btnFormating" href="http://asia.ensembl.org/Homo_sapiens/Tools/AssemblyConverter">Link</a></td>
    </tr>
    <tr>
      <td>ID History Converter</td>
      <td>Create the counterparts of a set of Ensembl IDs from a prior release</td>
      <td><a class="btnFormating" href="http://asia.ensembl.org/Homo_sapiens/Tools/IDMapper">Link</a></td>
    </tr>
    <tr>
      <td>Linkage Disequilibrium Calculator</td>
      <td>LD between variants can be calculated using genotypes from a particular population.</td>
      <td><a class="btnFormating" href="http://asia.ensembl.org/Homo_sapiens/Tools/LD">Link</a></td>
    </tr>
    <tr>
      <td>VCF to PED converter</td>
      <td>ICreate a linkage pedigree file (ped) and a marker information file from a vcf file by parsing so that they may be combined and fed into ld visualisation software like Haploview.</td>
      <td><a class="btnFormating" href="http://asia.ensembl.org/Homo_sapiens/Tools/VcftoPed">Link</a></td>
    </tr>
    <tr>
      <td>Data Slicer</td>
      <td>Obtain a portion of the data in a BAM or VCF file.</td>
      <td><a class="btnFormating" href="http://asia.ensembl.org/Homo_sapiens/Tools/DataSlicer">Link</a></td>
    </tr>
    <tr>
      <td>BioMart</td>
      <td>Its user-friendly web-based tool enables data extraction without the need for programming expertise or a thorough understanding of the underlying database structure.</td>
      <td><a class="btnFormating" href="http://asia.ensembl.org/biomart/martview">Link</a></td>
    </tr>
    <tr>
      <td>ID Mapping</td>
      <td>The ID Mapping service can map between the identifiers used in one database, to the identifiers of another, e.g., from UniProt to Ensembl, or to PomBase, etc.</td>
      <td><a class="btnFormating" href="https://www.uniprot.org/id-mapping">Link</a></td>
    </tr>
    <tr>
      <td>IEDB</td>
      <td>NIAID has financed the Immune Epitope Database (IEDB), a free online tool. In the context of infectious disease, allergies, autoimmunity, and transplantation, it compiles experimental data on antibody and T cell epitopes examined in non-human primates, humans, and other animal species. Additionally, the IEDB provides resources for epitope analysis and prediction.</td>
      <td><a class="btnFormating" href="https://www.iedb.org/">Link</a></td>
    </tr>
    <tr>
      <td>DDBJ</td>
      <td>Science is advanced through the sharing and analysis of data from life science researches provided by the Bioinformation and DDBJ Center.</td>
      <td><a class="btnFormating" href="https://www.ddbj.nig.ac.jp/index-e.html">Link</a></td>
    </tr>
    <tr>
      <td>Table Browser</td>
      <td>Utilizing the Genome Browser annotation track database, this programme is used to access and export data.</td>
      <td><a class="btnFormating" href="https://genome.ucsc.edu/cgi-bin/hgTables?hgsid=1446003591_QxaawWZEU8xuvs829dSuPR2rU6gF">Link</a></td>
    </tr>
  </tbody>
</table>
