<table>
  <thead>
    <tr>
      <th>Tool Name</th>
      <th>Description</th>
      <th>Link</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>Consurf Server</td>
      <td>Consurf Server uses a Bayesian inference method to calculate the evolutionary conservation of each residue in the protein.The conservation scores are then mapped onto the protein structure to identify the conserved regions.</td>
      <td><a class="btnFormating" href="https://consurf.tau.ac.il/consurf_index.php">Link</a></td>
    </tr>
    <tr>
      <td>ID History Converter</td>
      <td>ID History Converter supports a wide range of identifier formats, including gene symbols, Ensembl IDs, UniProt IDs, RefSeq IDs, and many others. It also supports conversion between different versions of the same identifier, allowing users to track changes in gene and protein annotation over time.</td>
      <td><a class="btnFormating" href="http://asia.ensembl.org/Homo_sapiens/Tools/IDMapper">Link</a></td>
    </tr>
    <tr>
      <td>Linkage Disequilibrium Calculator</td>
      <td>A Linkage Disequilibrium (LD) calculator is a tool used in population genetics to estimate the degree of association or correlation between alleles at two or more loci. LD is a measure of non-random association between alleles at different loci, and is influenced by factors such as genetic drift, mutation, migration, and natural selection.</td>
      <td><a class="btnFormating" href="http://asia.ensembl.org/Homo_sapiens/Tools/LD">Link</a></td>
    </tr>
    <tr>
      <td>Blat</td>
      <td>BLAT (BLAST-Like Alignment Tool) is a sequence similarity search algorithm and software tool used for aligning DNA or protein sequences to a reference genome or database. </td>
      <td><a class="btnFormating" href="https://genome.ucsc.edu/cgi-bin/hgBlat">Link</a></td>
    </tr>
    <tr>
      <td> MEGA X </td>
      <td>MEGA X offers a wide range of features and tools for analyzing DNA and protein sequences, including alignment, phylogenetic inference, evolutionary rate analysis, and molecular dating. </td>
      <td><a class="btnFormating" href="https://www.megasoftware.net/">Link</a></td>
    </tr>
    <tr>
      <td>GATK</td>
      <td>GATK includes a suite of tools and algorithms for processing genomic data, including tools for read mapping, quality control, base recalibration, variant calling, and annotation. </td>
      <td><a class="btnFormating" href="https://gatk.broadinstitute.org/hc/en-us">Link</a></td>
    </tr>
    <tr>
      <td>Tophat</td>
      <td>Tophat works by first aligning the RNA sequencing reads to the genome using a fast splice-junction mapper called Bowtie. It then identifies splice junctions from the mapped reads and constructs a transcriptome assembly based on these junctions. </td>
      <td><a class="btnFormating" href="https://ccb.jhu.edu/software/tophat/index.shtml">Link</a></td>
    </tr>
    <tr>
      <td>Prokka</td>
      <td>Prokka takes as input a bacterial or archaeal genome sequence in FASTA format and automatically annotates genes, coding sequences (CDSs), transfer RNAs (tRNAs), ribosomal RNAs (rRNAs), and other features. </td>
      <td><a class="btnFormating" href="https://kbase.us/applist/apps/ProkkaAnnotation/annotate_contigs/release?gclid=Cj0KCQiA9YugBhCZARIsAACXxeLB3NOdItUJWnRsK9OvidxwoEAmgEYMZ0ql4OYyeXKXqx5ZfVVB_YEaAkUSEALw_wcB">Link</a></td>
    </tr>
    <tr>
      <td>SAMtools</td>
      <td>SAMtools is a software package for working with SAM (Sequence Alignment/Map) format files. It can be used for read alignment, variant calling, and other DNA analysis tasks. </td>
      <td><a class="btnFormating" href="https://samtools.sourceforge.net/">Link</a></td>
    </tr>
    <tr>
      <td>Clustal</td>
      <td>Clustal is a multiple sequence alignment tool that aligns DNA or protein sequences to determine homology.</td>
      <td><a class="btnFormating" href="https://www.ebi.ac.uk/Tools/msa/clustalo/">Link</a></td>
    </tr>
    <tr>
      <td>Primer3</td>
      <td>This tool is used for designing PCR primers that can be used for DNA amplification.</td>
      <td><a class="btnFormating" href="https://primer3.ut.ee/">Link</a></td>
    </tr>
    <tr>
      <td>T-Coffee</td>
      <td>This tool performs multiple sequence alignment of DNA or protein sequences.</td>
      <td><a class="btnFormating" href="http://www.phylogeny.fr/one_task.cgi?task_type=tcoffee">Link</a></td>
    </tr>
    <tr>
      <td>MUSCLE</td>
      <td>MUSCLE (Multiple Sequence Comparison by Log-Expectation) is a software program that performs multiple sequence alignment of DNA or protein sequences.</td>
      <td><a class="btnFormating" href="http://www.phylogeny.fr/one_task.cgi?task_type=muscle">Link</a></td>
    </tr>
    <tr>
      <td>ProbCons</td>
      <td>ProbCons is a software program for multiple sequence alignment of DNA or protein sequences. It uses a probabilistic consistency algorithm that can handle large datasets and produce high-quality alignments.</td>
      <td><a class="btnFormating" href="http://www.phylogeny.fr/one_task.cgi?task_type=probcons">Link</a></td>
    </tr>
    <tr>
      <td>Translate tool</td>
      <td>This tool translates DNA sequences to their corresponding protein sequences using the standard genetic code.</td>
      <td><a class="btnFormating" href="https://web.expasy.org/translate/">Link</a></td>
    </tr>
    <tr>
      <td>Reverse Translate tool</td>
      <td>This tool generates all possible DNA sequences that encode a given protein sequence.</td>
      <td><a class="btnFormating" href="https://www.bioinformatics.org/sms2/rev_trans.html#:~:text=Reverse%20Translate%20accepts%20a%20protein,amino%20acid%20is%20also%20returned.">Link</a></td>
    </tr>
    <tr>
      <td>Restriction enzyme tool</td>
      <td>This tool helps to identify potential restriction enzyme sites in a DNA sequence.</td>
      <td><a class="btnFormating" href="http://nc2.neb.com/NEBcutter2/">Link</a></td>
    </tr>
    <tr>
      <td>Codon Usage Table tool</td>
      <td>This tool generates a codon usage table for a given DNA sequence or set of sequences.</td>
      <td><a class="btnFormating" href="https://www.bioinformatics.org/sms2/codon_usage.html">Link</a></td>
    </tr>
    <tr>
      <td>Pattern search tool</td>
      <td>This tool searches for a given pattern or regular expression in a DNA sequence.</td>
      <td><a class="btnFormating" href="https://www.bioinformatics.org/sms2/protein_pattern.html">Link</a></td>
    </tr>
    <tr>
      <td>ORF Finder tool</td>
      <td>This tool identifies open reading frames (ORFs) in a DNA sequence and predicts the corresponding amino acid sequences</td>
      <td><a class="btnFormating" href="https://www.bioinformatics.org/sms2/orf_find.html#:~:text=ORF%20Finder%20searches%20for%20open,alphabet%20and%20several%20genetic%20codes.">Link</a></td>
    </tr>
    <tr>
      <td>DNA Stats tool</td>
      <td>This tool provides basic statistics for a DNA sequence, including the length, GC content, and nucleotide composition.</td>
      <td><a class="btnFormating" href="https://www.bioinformatics.org/sms2/dna_stats.html#:~:text=DNA%20Stats&text=DNA%20Stats%20returns%20the%20number,results%20obtained%20for%20different%20sequences.">Link</a></td>
    </tr>
    <tr>
      <td>InterProScan tool</td>
      <td>This tool searches protein and DNA sequences against several databases to identify functional domains and motifs.</td>
      <td><a class="btnFormating" href="https://www.ebi.ac.uk/interpro/">Link</a></td>
    </tr>
    <tr>
      <td>Trimest</td>
      <td>This tool predicts the locations of repetitive DNA elements in a DNA sequence.</td>
      <td><a class="btnFormating" href="https://www.bioinformatics.nl/cgi-bin/emboss/trimest">Link</a></td>
    </tr>
    <tr>
      <td>Palindrome</td>
      <td>This tool identifies potential palindromic sequences (sequences that read the same forwards and backwards) in a DNA sequence.</td>
      <td><a class="btnFormating" href="https://www.bioinformatics.nl/cgi-bin/emboss/palindrome">Link</a></td>
    </tr>
    <tr>
      <td>Tranalign</td>
      <td>This tool performs multiple sequence alignment of DNA or protein sequences using a pairwise alignment approach.</td>
      <td><a class="btnFormating" href="https://www.bioinformatics.nl/cgi-bin/emboss/tranalign">Link</a></td>
  </tbody>
</table>
